high_card.rb

```diff
@@ -3,3 +3,30 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+deck = []
+results = []
+
+suits.each do |suit|
+    ranks.each do |rank|
+      deck.push([rank,suit])
+    end
+end
```

Comment:
I would recommend using a hash instead of an array. Sometimes indexing by integer can be potentially a little cryptic. Often times when dealing with more complex objects like a card its better to use hash because were able to semantically index with hashes.